src/main/resources/base/console_backend.capnp

```diff
@@ -452,6 +452,23 @@ struct AutoSurveyRequest {
     request @0 :Void = void;
 }
 
+struct Position {
+    lat @0: Float64;
+    lon @1: Float64;
+    alt @2: Float64;
+}
+
+struct NtripConnect {
+    url @0 :Text;
+    username @1 :Text;
+    password @2 :Text;
+    epoch @3 :UInt64;
+    position :union {
```

Comment:
maybe better / less space to take some out of bounds value as to be null rather constructing like this.

Reply:
don't know enough bout capnp to comment